prime-router/settings/organizations-prod.yml

```diff
@@ -131,7 +131,6 @@
       deidentify: false
```

Comment:
So, you really can't just load organizations-prod.yml into Prod any more - the "source of truth" is now the settings table in the database, not this file.  If you just load this file, you may overwrite other tweaks someone has done in Prod.

For a new process, the thought was that you put little mini yamlito chunks into `settings/prod/00xx-some-useful-name.yml`, so that those production changes get reviewed.   Then, you load those yamlitos into Prod using the settings API.    I think the only realistic and safe way to create the yamlitos is to first download the desired setting from Prod, change it, check it into a PR for second-set-of-eyes, then upload it.

That whole process is a work in progress, and admittedly ungainly -- feel free to propose a better idea.   The thought was, there's no way to test these changes, so its important to have two eyes on them, so might as well use the PR process.

Reply:
I can do that as well. I was adding them to `organizations-prod.yml` so we had a platonic ideal, and then do the updates via the CLI.

Reply:
Yeah, I just think that's dangerous now -- the platonic ideal should only come from `./prime <organization|receiver|sender> get`